src/stack.js

```diff
@@ -1,5 +1,29 @@
 'use strict'
 
 export default class Stack {
-  // your code here
+  constructor(){
+    this.size = 0
+    this.value = [ ]
+  }
+
+  push(element){
+    this.value.unshift(element)
+    this.size++
+  }
+
+  pop(){
+    return this.value.pop()
+  }
```

Comment:
From the specs:
>pop, which removes the most recently added element 

This `pop(`) function that you've written will not return the most recently pushed element.  It will return the last element of `this.value`, but since your `push()` function adds elements to the beginning of the `this.value()`, the most recent elements are actually at the beginning of the array.